scripts/table_diff/changes/changes.go

```diff
@@ -47,13 +48,35 @@ func parseColumnChange(line string) (name string, dataType string, pk bool) {
 	return cleanName, result["dataType"], result["name"] != cleanName
 }
 
+func parsePKChange(line string) (names []string) {
+	for _, match := range pkRegex.FindAllStringSubmatch(line, -1) {
```

Comment:
Are we expecting to match the regex multiple times in a single line? If not can we use `FindStringSubmatch`?

Reply:
yes, sure!